daemon/logger/awslogs/cloudwatchlogs_test.go

```diff
@@ -404,40 +404,16 @@ func TestLogBlocking(t *testing.T) {
 	}
 }
 
-func TestLogNonBlockingBufferEmpty(t *testing.T) {
+func TestLogBufferEmpty(t *testing.T) {
 	mockClient := &mockClient{}
 	stream := &logStream{
 		client:         mockClient,
 		messages:       make(chan *logger.Message, 1),
-		logNonBlocking: true,
 	}
 	err := stream.Log(&logger.Message{})
 	assert.NilError(t, err)
 }
 
-func TestLogNonBlockingBufferFull(t *testing.T) {
```

Comment:
@corhere so this test tests the buggy behavior... so I just removed it..

(a bug that has a test to check that the bug exists xD!)